### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kkennethlee**
- Extracted User Event Logs from Docker MySQL

- Styled it in more presentable format

- Sort by ID function with click event binding

- Loader animation to appear when making a call, and disappear when the call is finished.

### CODE REVIEW

File: app/themes/user-admin/src/app/shared/users.service.ts [hunk 1]
```diff
@@ -224,6 +224,8 @@ export class UsersService {
     
         let options = this._getHeaderOptions();
 
+        console.log("URL", url)
```
Comment (saarmstrong) [inline app/themes/user-admin/src/app/shared/users.service.ts]: Please delete this debug statement.

File: app/themes/user-admin/src/app/user-event-log/user-event-log.component.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+  //Variable for the loading animation.
+  private wait: boolean;
+
+  //initialize the subscription
+  private events$: Subscription;
+  private events: Event[];
+
+  //Parameters of getUserEvents()
+  private userID = '1';
```
Comment (saarmstrong) [inline app/themes/user-admin/src/app/user-event-log/user-event-log.component.ts]: This ID should not be hardcoded, the User ID should come from the router itself (like a query string value).

Check out these docs for more info:
- https://angular.io/docs/ts/latest/guide/router.html#!#route-parameters
- http://stackoverflow.com/a/40276279/6357846

If you need an working example, take a look at this the `ngOnInit` method in the user-edit component itself: https://github.com/Erdiko/user-admin/blob/master/app/themes/user-admin/src/app/user-edit/user-edit.component.ts#L48

File: app/themes/user-admin/src/app/user-event-log/user-event-log.component.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                                <alert type="warning">Sorry, no user events were found. Please try again.</alert>
+                            </td>
+                        </tr>
+                    </tbody>
+
+                    <tbody *ngIf="!wait && events.length > 0">
+                        <tr *ngFor="let event of events">
+                            <td> {{event.id}} </td>
+                            <td> {{event.event}} </td>
+                            <td> {{event.event_data}} </td>
```
Comment (saarmstrong) [inline app/themes/user-admin/src/app/user-event-log/user-event-log.component.html]: useradmin_and_3__node_and_user-admin_ _working_copy__kkennethlee-develop_ _2_changed_files__and_2__mywu__bash_

Please render the event_data objects (if its an object) via json. Here is how you can accomplish this:

`{{ event.event_data | json }}`

Comment (kkennethlee) [inline app/themes/user-admin/src/app/user-event-log/user-event-log.component.html]: That's strange, I never once had an [object Object] output on my end.

File: app/themes/user-admin/src/app/shared/users.service.ts (hunk 1 above)
Comment (kkennethlee) [inline app/themes/user-admin/src/app/shared/users.service.ts]: Done.

File: app/themes/user-admin/src/app/user-event-log/user-event-log.component.ts (hunk 2 above)
Comment (kkennethlee) [inline app/themes/user-admin/src/app/user-event-log/user-event-log.component.ts]: Fixed.

**Comment by kkennethlee** [comment]
Below are my final output that address the changes requested:

User Don Draper:
screen shot 2017-03-05 at 12 51 38 am

User Beef:
screen shot 2017-03-05 at 12 52 08 am

**Pull request merged**